test/WebSites/BasicWebSite/Startup.cs

```diff
@@ -32,6 +41,16 @@ public void ConfigureServices(IServiceCollection services)
                 .SetCompatibilityVersion(CompatibilityVersion.Latest)
                 .AddXmlDataContractSerializerFormatters();
 
+            if (!_hostingEnvironment.IsDevelopment())
```

Comment:
This would need to be unconditional if watching were enabled in `CompatibilityVersion.Latest`.